python/lsst/ip/isr/isrTask.py

```diff
@@ -597,7 +599,15 @@ def getIsrExposure(self, dataRef, datasetType, immediate=True):
         try:
             exp = dataRef.get(datasetType, immediate=immediate)
         except Exception, e:
-            raise RuntimeError("Unable to retrieve %s for %s: %s" % (datasetType, dataRef.dataId, e))
+            try:
+                exp = dataRef.get(datasetType, filter=self.config.fallbackFilterName, immediate=immediate)
+                self.log.warn("Using fallback calibration from filter %s" % self.config.fallbackFilterName)
+            except:
+                extraMessage = ''
+                if not self.config.fallbackFilterName:
+                    extraMessage += 'And no fallback filter specified'
+                raise RuntimeError("Unable to retrieve %s for %s %s: %s" % (datasetType, dataRef.dataId,
+                                                                            extraMessage, e))
```

Comment:
I think this will lead to a space between the text and the colon if no `extraMessage` is provided. I'd rather it didn't but your choice.